exonum-java-binding/core/rust/ejb-app/start_qa_node.sh

```diff
@@ -69,27 +63,23 @@ do
 done
 
 header "GENERATE COMMON CONFIG"
-cargo run -- generate-template --validators-count $node_count testnet/common.toml
+cargo run -- generate-template --validators-count $node_count testnet/common.toml \
```

Comment:
why does it use `cargo`, not `ejb-app`?

Reply:
Because you can't guarantee the binary ejb-app in `target` directory would be valid (linked against right libraries). When running integration tests, we build a special version of libjava_bindings, which can't be used with the `ejb-app` binary build for packaging, for example. So we run cargo just to ensure if the right library is provided and recompile otherwise. After running this script for the first time you can, however, run the binary directly from target.

Reply:
> running integration tests, we build a special version of libjava_bindings

Rust unit tests (no-resource-manager)? Otherwise I don't understand.

---

Shall @dip56 create a separate script which uses an **installed** application? And this be kept for development purposes?

Reply:
Oh, yes, unit tests without resource manager. Created ECR-3063 for end-users support. These scripts will be used for developing purposes only.